tempesta_fw/procfs.c

```diff
@@ -151,18 +151,22 @@ tfw_srvstats_seq_show(struct seq_file *seq, void *off)
 
 	int i, ret;
 	TfwServer *srv = seq->private;
-	Percentile pcntl[] = { {1}, {50}, {75}, {90}, {95}, {99} };
+	Percentile prcntl[] = {
+		{tfw_apm_prcntl_ith[0]}, {tfw_apm_prcntl_ith[1]},
+		{tfw_apm_prcntl_ith[2]}, {tfw_apm_prcntl_ith[3]},
+		{tfw_apm_prcntl_ith[4]}, {tfw_apm_prcntl_ith[5]},
+	};
 
-	tfw_apm_calc_user(srv->apm, pcntl, ARRAY_SIZE(pcntl));
+	tfw_apm_stats(srv->apm, prcntl, ARRAY_SIZE(prcntl));
 
-	SPRNE("Minimal response time\t\t", pcntl[0].val);
-	SPRNE("Maximum response time\t\t", pcntl[5].val);
-	SPRNE("Median  response time\t\t", pcntl[2].val);
+	SPRNE("Minimal response time\t\t", prcntl[0].val);
+	SPRNE("Maximum response time\t\t", prcntl[5].val);
+	SPRNE("Median  response time\t\t", prcntl[2].val);
```

Comment:
Maybe it'll be useful to swap those lines to show stat in `Min. Median Max.` order ?